source/browseMode.py

```diff
@@ -1576,6 +1592,7 @@ def _getTableCellAt(self,tableID,startPos,row,column):
 		@type column: int
 		@returns: the table cell's position in the document
 		@rtype: L{textInfos.TextInfo}
+		@raises: L{HiddenCellFound} if the cell it founds is hidden, allowing the caller to try a different cell.

```

Comment:
I'm concerned that we might not actually be able to tell the difference between a hidden cell and a cell which is out of bounds. Chrome doesn't render aria-hidden content in the tree at all, for example, and Gecko will probably do the same (eventually). Is there any reason we shouldn't just use LookupError here?